extensions/vscode/src/multiStepInputs/newConnectCredential.ts

```diff
@@ -352,12 +356,34 @@ export async function newConnectCredential(
       const tokenActivator = new ConnectAuthTokenActivator(serverUrl, viewId);
       await tokenActivator.initialize();
 
-      // Activate the token
-      const result = await tokenActivator.activateToken();
+      const resp = await input.showInfoMessage<
```

Comment:
If we click the back button during the next step `INPUT_CRED_NAME` what currently happens?

Do we need to utilize `skipStepHistory` like some of the `newConnectCloudCredentials` steps do to avoid this step being backed into?